draft-trammell-taps-post-sockets.md

```diff
@@ -508,27 +508,43 @@ protocol stack information, and, per {{I-D.pauly-taps-transport-security}},
 cryptographic identities (certificates and associated private keys) bound to
 this endpoint.
 
-## Policy Context {#PolicyContext}
-
-The Policy Context describes preferences for, and restrictions on, how to
-configure Transients to support communication between a Local and a Remote
-over one or more Paths between endpoints.
-For instance, an application may require, or prefer to use, certain features
-(see {{I-D.ietf-taps-transports}}) of the transport protocol stacks used by
-the Transients underlying the Carrier.
-Alternatively, it might also prefer Paths over one interface to those over
-another (e.g., WiFi access over LTE when roaming on a foreign LTE network,
-due to cost).
-
-These policies are expressed in the Policy Context(s) that are bound to the
-Association.
-Multiple policy contexts can be active at once.
-For example, a system Policy Context can express the administrative preferences
-around network interface and protocol selection, while an application Policy Context
-expresses preferences for use of different transport services.
-Expression of policy contexts and the resolution of conflicts among Policy
-Contexts is currently implementation-specific (the Policy API in the NEAT
-architecture {{NEAT}} provides an example of how this can be done).
+## Configuration {#Configuration}
+
+A Configuration encapsulates an application's preferences around Path
+selection and protocol options.
```

Comment:
Could you add something such as, "For example, a Configuration may contain default trust model information used when establishing connections."

Or it could go below when you describe how configs are used. (One TLS config might use pinning, another may not.)

Reply:
Added a section at the end!